src/component/signUp/SignUp.jsx

```diff
@@ -0,0 +1,95 @@
+import { useState, useContext } from 'react';
+import SimpleSnackbar from '../snack/Snack';

```

Comment:
SimpleSnackbar is a component, why are you importing it before react-router-dom?